ios/api.md

```diff
@@ -464,119 +464,77 @@ do {
-
-5. Select all the files under `generated-src` folder of your application's root folder and add them to your project.
+```
 
-6. Next, set the bridging header for Swift in your project settings. Double-click your project name in the Xcode Project Navigator, choose the Build Settings tab and search for  `Objective-C Bridging Header`. Enter `generated-src/Bridging_Header.h`
+Run `pod install --repo-update` and then add the `generated-src` folder and `awsconfiguration.json` file to your project **(File->Add Files to ..->Add)** and then build your project, ensuring there are no issues.
 
-	This is needed because the AWS generated code has some Objective-C code which requires bridging to be used for Swift.
+Next, set the bridging header for Swift in your project settings. Double-click your project name in the Xcode Project Navigator, choose the Build Settings tab and search for  `Objective-C Bridging Header`. Enter `generated-src/Bridging_Header.h`
 
-	> If you already have a bridging header in your app, you can just append an extra line to it: `#import "AWSApiGatewayBridge.h"` instead of above step.
+This is needed because the AWS generated code has some Objective-C code which requires bridging to be used for Swift. If you already have a bridging header in your app, you can just append an extra line to it: `#import "AWSApiGatewayBridge.h"` instead of above step.
+{: .callout .callout--action}
 
-7. Use the files generated by CLI to determine the client name of your API. In the `generated-src` folder, files ending with name `*Client.swift` are the names of your client (without .swift extension).
+The generated files determine the name of your client when making API calls. In the `generated-src` folder, files ending with name `*Client.swift` are the names of your client (without .swift extension). The path of the client code file is: 
 
-	The path of the client code file is `./generated-src/YOUR_API_RESOURCE_NAME+YOUR_APP_NAME+Client.swift`.
+**./generated-src/YOUR_API_RESOURCE_NAME+YOUR_APP_NAME+Client.swift**
 
-	So, for an app named `useamplify` with an API resource named `xyz123`, the path of the code file might be `./generated-src/xyz123useamplifyabcdClient.swift`. The API client name would be `xyz123useamplifyabcdClient`.
+So, for an app named `useamplify` with an API resource named `xyz123`, the path of the code file might be **./generated-src/xyz123useamplifyabcdClient.swift**. The API client name would be `xyz123useamplifyabcdClient` and you would use it in your code with `xyz123useamplifyabcdClient.registerClient()` and `xyz123useamplifyabcdClient.client()`.
 
-	- Find the resource name of your API by running `amplify status`.
-	- Copy your API client name to use when invoking the API in the following step.
 
+Find the resource name of your API by running `amplify status`. Copy your API client name to use when invoking the API in the following sections.
 
-8. Invoke a Cloud Logic API.
+#### IAM authorization
```

Comment:
Looks like you mostly lower case authorization, but in the above title is “Setup Your Backend”. Maybe standardize on one method for the titles?

Reply:
Is there a proper titling that we should do here? If so let's go through all of them and update.